tools/windows/DatadogAgentInstaller/CustomActions/Flare.cs

```diff
@@ -68,7 +68,8 @@ public void Send(string email)
                 { "version", agentVersion },
                 { "log", log },
                 { "email", email },
-                { "apikey", apikey }
+                { "apikey", apikey },
+                { "variant", "ng_windows_installer" }
```

Comment:
Not sure what a good name is here, or how we refer to it internally/externally - happy to rename as needed.